### DISCUSSION TITLE
How to annotate types to multi asset

### DISCUSSION THREAD

**Discussion by takerfume** [dagster-io/dagster#19211] [Q&A]
I would like to annotate type to multi asset.
How can I do?

**Comment by jamiedemaria** [comment]
Hi @takerfume  I'm going to need a little more information to help you out. Can you describe what you're trying to do in a bit more detail. Some code snippets of what you have so far would also be helpful

**Reply by takerfume** [reply]
sorry for my ambiguous question.
I was mistaken and I am thinking now this question is not important.
